rend3-gltf/Cargo.toml

```diff
@@ -22,7 +22,7 @@ ddsfile = { version = "0.5", optional = true }
 float-ord = "0.3.2"
 glam = "0.20.0"
 gltf = { version = "1.0", default-features = false, features = ["KHR_lights_punctual", "KHR_texture_transform", "KHR_materials_unlit", "extras", "names", "utils"] }
-image = { version = "0.23", default-features = false }
+image = { version = "0.23", features = ["jpeg"] }
```

Comment:
Not sure if we want to force the `jpeg` feature on downstream users. Last time I checked with @cwfitzgerald (please correct me if I'm wrong!). I think they mentioned this is something that should be specified on the user's crate. In this case, the change should be made to the animation example's Cargo.toml.

Reply:
Fixed